e2e/steps_file.js

```diff
@@ -146,9 +146,10 @@ module.exports = function () {
       await this.completeEvent('Save and continue');
       await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
       uploadDocumentsEventPage.selectSocialWorkChronologyToFollow(config.testFile);
```

Comment:
I know it's not your code, but is it correct we are selecting to follow and passing file? How does it not fail the validation.

Reply:
The method does not use the file param. Removing. It does not fail validation as we no longer validate on 'to follow' as per AC update.